en/concept/flight_modes.md

```diff
@@ -41,7 +41,7 @@ The sections below provide an overview of the modes, followed by a [flight mode
   * **Multirotors:**  The multirotor hovers / loiters at the current position and altitude.
 * **AUTO_RTL** (Return to Land)
   * **Fixed wing aircraft:** The aircraft returns to the home position and loiters in a circle above the home position. 
-  * **Multirotors:** The multirotor returns in a straight line on the current altitude (if the current altitude is higher than the home position + loiter altitude) or on the loiter altitude (if the loiter altitude is higher than the current altitude), then lands automatically.
+  * **Multirotors:** The multirotor returns in a straight line on the current altitude (if the current altitude is higher than the home position + <span id="RTL_RETURN_ALT"></span>[RTL_RETURN_ALT](../advanced_config/parameter_reference.md#RTL_RETURN_ALT)) or on the <span id="RTL_RETURN_ALT"></span>[RTL_RETURN_ALT](../advanced_config/parameter_reference.md#RTL_RETURN_ALT) (if the <span id="RTL_RETURN_ALT"></span>[RTL_RETURN_ALT](../advanced_config/parameter_reference.md#RTL_RETURN_ALT) is higher than the current altitude), then lands automatically.
```

Comment:
1. You don't need the "span"s here - you only need them if you expect to link to this point. 
2. FW and MC have the same behaviour with respect to RTL_RETURN_ALT. The only real difference is that the default behaviour on return for FW is to circle rather than land. 
3. Good to link to the user guide.

How about for this section:
 * **[AUTO_RTL](https://docs.px4.io/en/flight_modes/rtl.html)** (Return): The vehicle ascends to altitude `RTL_RETURN_ALT` (or maintains altitude if already at a greater height) and returns to the home position.  By default a multicopter will land at the home position, while a fixed-wing vehicle will loiter in a circle above it.